src/Components/OATPropertyEditor/Internal/TemplateList.tsx

```diff
@@ -201,12 +232,12 @@ export const TemplateList = ({
                     : () => onDragEnterExternalItem(0)
             }
         >
-            {state &&
-                templates &&
-                templates.length > 0 &&
-                templates.map((item, i) => (
+            {oatPageState &&
+                oatPageState.currentOntologyTemplates &&
+                oatPageState.currentOntologyTemplates.length > 0 &&
+                oatPageState.currentOntologyTemplates.map((item, i) => (
```

Comment:
I think this would read easier as `oatPateState?.currentOntologyTemplates?.map(...`

Reply:
Agreed. I am not going to make chnages to templates for now but it definitely needs a full revamp. If/when we get to templates I'll definitely do this cleanup.